backend/src/main/java/gov/cdc/usds/simplereport/api/model/accountrequest/AccountRequest.java

```diff
@@ -61,17 +62,19 @@ public String getTemplateName() {
     variableMap.put("workPhoneNumber", workPhoneNumber);
     variableMap.put("cellPhoneNumber", cellPhoneNumber);
     variableMap.put("mailingAddress1", mailingAddress1);
-    variableMap.put("aptSuiteOther", aptSuiteOther);
-    variableMap.put("aptFloorSuiteNo", aptFloorSuiteNo);
+    variableMap.put("streetAddress2", streetAddress2);
     variableMap.put("city", city);
     variableMap.put("state", state);
     variableMap.put("zip", zip);
     variableMap.put("county", county);
     variableMap.put("facilityType", facilityType);
+    variableMap.put("facilityTypeOther", facilityTypeOther);
     variableMap.put("organizationName", organizationName);
     variableMap.put("facilityName", facilityName);
     variableMap.put("cliaNumber", cliaNumber);
     variableMap.put("testingDevices", testingDevices);
+    variableMap.put("testingDevicesOther", testingDevicesOther);
+    variableMap.put("defaultTestDevice", defaultTestDevice);
     variableMap.put("accessDevices", accessDevices);
     variableMap.put("browsers", browsers);
```

Comment:
We also have an ["other browser" new field](https://github.com/CDCgov/prime-simplereport-site/pull/103/files#diff-67012e167bb9599500bd2ec11dad8bce9556b42c94ff5d6cc8d97154a052532eR254). Could you add that too?

Reply:
Yep! I'll add it.